### PR TITLE
New package: Gaming.Intelligence version 0.0.2.77

### LINKED ISSUE #144938: [Package Request]: Add MSI Gaming Intelligence Software
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.msi.com/support/technical_details/MNT_SW_MSI_Preloaded_Software#txt01
Publisher: MSI
Package Name: MSI Gaming Intelligence
Description: Used to control MSI Gaming Monitors
Package Version: 0.0.2.77
Installer URL: https://download.msi.com/uti_exe/monitor/Gaming-Intelligence-x64.zip

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

* Resolve #144938

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/144958)

### CODE REVIEW

File: manifests/g/Gaming/Intelligence/0.0.2.77/Gaming.Intelligence.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Gaming.Intelligence
+PackageVersion: 0.0.2.77
+InstallerType: zip
+NestedInstallerType: exe
```
Comment (vikingnope) [inline manifests/g/Gaming/Intelligence/0.0.2.77/Gaming.Intelligence.installer.yaml:7]:
```suggestion
InstallModes:
- interactive
- silent
- silentWithProgress
InstallerSwitches:
  Silent: /VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP- /NoAutoStart
  SilentWithProgress: /SILENT /SUPPRESSMSGBOXES /NORESTART /SP- /NoAutoStart
NestedInstallerType: exe
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 870.)

**Comment by vikingnope** [comment]
@stephengillie, can this be waived as well, guessing same issue here

File: manifests/g/Gaming/Intelligence/0.0.2.77/Gaming.Intelligence.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Gaming.Intelligence
+PackageVersion: 0.0.2.77
+InstallerType: zip
+NestedInstallerType: exe
+NestedInstallerFiles:
+- RelativeFilePath: setup.exe
+Installers:
```
Comment (stephengillie) [inline manifests/g/Gaming/Intelligence/0.0.2.77/Gaming.Intelligence.installer.yaml:10]:
```suggestion
InstallerSwitches:
  Silent: /S /v /qn
Installers:
```

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> 2024-03-18 17:07:01.731 [CLI ] Package hash verification failed. SHA256 in manifest [2f116aea817d6414fd3687ccf50f39cefeba6a6d8c516b42975d3baffd44aba4] does not match download [94b3380502d853e1bc11fade49e66d9778b45c90cf55206816688d4f3c0ef71c]

(Automated response - build 870.)

File: manifests/g/Gaming/Intelligence/0.0.2.77/Gaming.Intelligence.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: Gaming.Intelligence
+PackageVersion: 0.0.2.77
+InstallerType: zip
+NestedInstallerType: exe
+NestedInstallerFiles:
+- RelativeFilePath: setup.exe
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.msi.com/uti_exe/monitor/Gaming-Intelligence-x64.zip
+  InstallerSha256: 2F116AEA817D6414FD3687CCF50F39CEFEBA6A6D8C516B42975D3BAFFD44ABA4
```
Comment (vikingnope) [inline manifests/g/Gaming/Intelligence/0.0.2.77/Gaming.Intelligence.installer.yaml:13]:
```suggestion
  InstallerSha256: 94B3380502D853E1BC11FADE49E66D9778B45C90CF55206816688D4F3C0EF71C
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installer Verification Analysis Context Information: 
 Specified hash doesn't match.
 Actual hash:   2F116AEA817D6414FD3687CCF50F39CEFEBA6A6D8C516B42975D3BAFFD44ABA4
 Expected hash: 94b3380502d853e1bc11fade49e66d9778b45c90cf55206816688d4f3c0ef71c
 
 3/19/2024 8:19:58 AM
 
 ****************************************************************************************
 Installer Validation
 ****************************************************************************************
 

(Automated response - build 870.)

File: manifests/g/Gaming/Intelligence/0.0.2.77/Gaming.Intelligence.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+InstallerType: zip
+NestedInstallerType: exe
+NestedInstallerFiles:
+- RelativeFilePath: setup.exe
+InstallerSwitches:
+  Silent: /S /v /qn
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.msi.com/uti_exe/monitor/Gaming-Intelligence-x64.zip
+  InstallerSha256: 94B3380502D853E1BC11FADE49E66D9778B45C90CF55206816688D4F3C0EF71C
```
Comment (stephengillie) [inline manifests/g/Gaming/Intelligence/0.0.2.77/Gaming.Intelligence.installer.yaml:15]:
```suggestion

```

(Automated response - build 870.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**